collect_app/src/main/res/values/strings.xml

```diff
@@ -656,4 +656,10 @@
     <string name="sms_invalid_phone_number_description">Set a destination phone number for SMS submissions or change your transport type.</string>
     <string name="google_invalid_account_title">Google account missing</string>
     <string name="google_invalid_account_description">Please select a Google account or change your server type.</string>
+
+    <!-- Text for temporary notification of upcoming change. Please do not translate "current()". -->
+    <string name="current_predicate_warning">This question will not be compatible with future versions of ODK Collect because it uses current() in a choice filter. \n\nPlease make sure the person who made this form reads the details linked below and takes one of the recommended actions.</string>
```

Comment:
Please make sure the person who made this form reads the details below and takes action.